orca_base/src/orca_base.cpp

```diff
@@ -4,6 +4,35 @@
 #include "orca_msgs/Lights.h"
 #include "orca_msgs/Thrusters.h"
 
+// TODO move defines to yaml
+
```

Comment:
Huge thanks for the catch and the PR! These #defines got added back in -- can you remove them?